infra/resources/src/main/res/values/strings.xml

```diff
@@ -438,6 +438,7 @@
     <string name="id_pool_validation_pool_empty_message">There are no records on the device</string>
     <string name="id_pool_validation_action_close">Close</string>
     <string name="id_pool_validation_action_sync">Sync and retry</string>
+    <string name="id_pool_validation_action_continue">Ignore and Continue</string>
```

Comment:
Is this the final text or a placeholder?

Reply:
@luhmirin-s I have added the final text